docs/docs/unreachable.md

```diff
@@ -0,0 +1,90 @@
+
+---
+title: Unreachable Strategy
+---
+
+# Unreachable Strategy
+
+In order for Marathon to provide partition aware unreachable strategy support there are 2 high level events that must occur; 1) Mesos needs to communicate a task is unreachable and 2) Marathon must respond to that event if unresolved within a specified amount of time. Each of these events have configuration options and DCOS system defaults which are worth review in order to fully understand how and when an unreachable task will be managed by Marathon.
+
+## Apache Mesos Unreachable Strategies
+
+Apache Meso's ability to communicate a task / node is unreachable is controlled by 2 concepts; 1) mesos-agent health check and 2) node rate limiter. Regarding agent health checks, the mesos-master flags of control are:
```

Comment:
s/Meso's/Mesos's ?

Reply:
Bullets and numbered lists are my favorite things in documentation. Make things a lot easier to scan.